elasticsearch/utils/es_load_kibana_ui_objects

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash -e
+
+source "logging"
+
+set -euo pipefail
+
+if [ -n "${DEBUG:-}" -o $loglevelint = 7 ] ; then
+    set -x
+    curl_output() {
+        python -mjson.tool
+    }
+else
+    curl_output() {
+        cat > /dev/null 2>&1
+    }
+fi
+
+usage() {
+    error Usage: $0 user-name
+    error The kibana index for the given user must already exist
+}
+
+if [ -z "${1:-}" ] ; then
+    usage
+    exit 1
+fi
+
+kibindex=$( get_kibana_index_name "$1" )
+
+resp_code=$( QUERY="/$kibindex" es_util -XHEAD -w '%{response_code}' )
+
+if [ "$resp_code" != 200 ] ; then
+    error Could not find kibana index $kibindex for user $1: $resp_code
+    exit 1
+fi
+
+info Adding Kibana dashboards and other UI objects for user $1 index $kibindex
+
+INDEX_PATTERN=${INDEX_PATTERN:-project.*}
+INDEX_PATTERN_FILE=${INDEX_PATTERN_FILE:-/usr/share/elasticsearch/index_patterns/com.redhat.viaq-openshift.index-pattern.json}
```

Comment:
We should consider refactoring '/usr/share/elasticsearch/' into a var like ES_HOME.  If I ever get the reconciliation PR complete I think this will simply the impact and additional modifications we need to make.